sapirbitan/assignment1/ex2.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+#include <string>
+using namespace std;
+
+struct some_struct {
+	int num;
+	some_struct* s=NULL;
+};
+/* I am assuming each element in the linked list contains a pointer to the next element in the list,
+also, I am assuming my function gets the "head" of the linked list and 'k'	*/
+
+some_struct find_kth_element(some_struct first_element, int k) {
+	int num_of_elements = 1;
+	some_struct* current = &first_element;
+	//if k>num_of_elements the function will return the "head" of the list
+	some_struct kth=first_element;// the kth last element
```

Comment:
You should declare variables close to where they are used for readability. 'kth' is not used in the first while loop, please only declare it before the for loop where you're using it.